apps/admin_panel/assets/src/omg-exchange-rate-modal/createExchangeModal.js

```diff
@@ -231,30 +235,25 @@ class CreateExchangeRateModal extends Component {
       onlyOneWayExchange,
       oppositeExchangePair
     } = this.state
-
-    const forwardRate = _.round(toTokenRate / fromTokenRate, 3)
-    const backRate = _.round(1 / forwardRate, 3)
-
+    const forwardRate = +new BigNumber(numeral(toTokenRate).value()).dividedBy(numeral(fromTokenRate).value()).toFixed(8)
+    const backRate = new BigNumber(1).dividedBy(new BigNumber(numeral(forwardRate).value())).toFixed(8)
```

Comment:
`backRate` is not necessarily the inverse of `forwardRate` though, that's why we have separate pairs on the backend to support exchange rates that usually have a spread.

Reply:
![Screen Shot 2562-05-07 at 11 02 05 (2)](https://user-images.githubusercontent.com/29919550/57270965-a6b95280-70b7-11e9-9d4c-564c8d9cb832.png)

i guess backRate now is just the suggested(?) opposite rate, with zero spread... maybe we should add a 'spread' input?

Reply:
Ah since it's only a suggestion and can be edited without changing the forward rate, this should be fine.